### ISSUE TITLE
support spanner BatchCreateSessions

### ISSUE THREAD

**Issue by mr-salty** [googleapis/google-cloud-cpp-spanner#412]
spanner just added a [`BatchCreateSessions`](https://github.com/googleapis/googleapis/blob/4df4371ddaa2ee05fbece4eaab6d9effd27a71ee/google/spanner/v1/spanner.proto#L69-L79) method. Add support for it in `SpannerStub` and anywhere we mock it.

**Comment by coryan** [comment]
Will this change the public API?  If the answer is "no" or "probably not" then we should push this to the Beta milestone.

**Comment by coryan** [comment]
- [ ] We should start by breaking the dependency on google-cloud-cpp to get googleapis/cpp-cmakefiles (#414).
- [ ] We should update googleapis/cpp-cmakefiles to a newer version of the protos (googleapis/cpp-cmakefiles/issues#17).
- [ ] Then we should talk about adding the necessary functions to `SpannerStub` and `SpannerConnection` **once we start using them**, otherwise we are just adding dead code.

**Comment by mr-salty** [comment]
Yeah, Beta is probably fine, I made it Alpha because I wasn't sure if updating the protos would break anything (new pure virtual methods? or no?) and it would be good to have it available to experiment with.

**Comment by coryan** [comment]
FYI, the proto files are updated to include the `BatchCreateSessions` RPC.
